src/scripts/main.js

```diff
@@ -1 +1,90 @@
 'use strict';
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  // eslint-disable-next-line no-shadow
+  document.addEventListener('contextmenu', (event) => {
+    event.preventDefault();
+
+    resolve('Second promise was resolved');
+  });
+
+  // eslint-disable-next-line no-shadow
+  document.addEventListener('click', (event) => {
+    resolve('Second promise was resolved');
+  });
```

Comment:
Having multiple event listeners for the same event (click) in different promises can lead to unexpected behavior. Consider whether this is intentional and if not, refactor the code to avoid multiple listeners for the same event.